ospd/ospd.py

```diff
@@ -619,13 +620,17 @@ def sort_host_finished(
             scan_id, finished_hosts
         )
 
+    def set_target_progress(self, scan_id: str):
```

Comment:
Doesn't this method set the scan progress instead of the target progress? Currently I find the term target a bit confusing here. Could there be several targets in a scan?